www/js/blockly-api.js

```diff
@@ -95,9 +96,14 @@ function initApi(interpreter, scope) {
 /*-----HELPER FUNCTIONS -----*/
 
 function sendMotorCommand(command, pin, speed) {
-	$.post('send-command.php', {command: command, pin: pin, speed: Number(speed)}, function(response){
-		//writeToConsole(response);
-	});
+	$.post('http://localhost:5000/api/v1/sendcommand',

```

Comment:
I'm not sure why, but I had to stringify the whole payload for it to be sent as json.